avail-workbench/build.gradle

```diff
@@ -173,7 +148,7 @@ task osx {
 		}
 
 		exec {
-			def launcher = "$appDst/Contents/MacOS/JavaAppLauncher"
+			def launcher = appDst + "/Contents/MacOS/JavaAppLauncher"
```

Comment:
Why did you turn the interpolation into concatenation here? This definitely reads worse. Was it not working somehow?

Reply:
You added new code that uses interpolation just below in this same file, which makes it extra weird.

Reply:
The interpolation reads way better of course, but IntelliJ 2021.2 falsely flags it as a Groovy type error, which is annoying.  It runs either way.  IntelliJ also flags "jar { manifest ... }" as well, but there doesn't seem to be any workaround for that neglected piece of key infrastructure that *everybody who creates a jar* encounters.